components/common/FileInformation.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, useState } from 'react';
+import CrossIcon from '../../public/svg/CrossIcon';
+import EyeIcon from '../../public/svg/EyeIcon';
+import Tooltip from './Tooltip';
+
+interface FileInformationProps {
+  fileName: string;
+  fileType: string;
+  fileSize: string;
+  number: number;
+  crossIconClick: () => void;
+  viewIconClick: () => void;
+}
+
+const FileInformation: FC<FileInformationProps> = function ({
+  fileName,
+  fileType,
+  fileSize,
+  number = 1,
+  crossIconClick,
+  viewIconClick,
+}) {
+  const [showContent, setShowContent] = useState(false);
+  return (
+    <div className="flex items-center shadow-lg px-2 rounded-sm">
+      <p className="text-sm font-bold w-3 ">{`${number + 1}.`}</p>
+      <div className="flex items-center justify-between p-2 rounded-md min-w-full">
+        <p className="text-sm font-bold w-1/3  truncate">{fileName}</p>
+        <p className="text-sm font-bold text-center  ">{fileType}</p>
+        <div className="flex gap-x-2 items-center">
+          <Tooltip
```

Comment:
Move delete icon to the far right of the component (behind the file size display).

Reply:
done

Reply:
done